src/macros.rs

```diff
@@ -32,12 +32,13 @@
 #[macro_export(local_inner_macros)]
 macro_rules! log {
     (target: $target:expr, $lvl:expr, $($arg:tt)+) => ({
+        let target = $target;
         let lvl = $lvl;
-        if lvl <= $crate::STATIC_MAX_LEVEL && lvl <= $crate::max_level() {
+        if log_enabled!(target: target, lvl) {
```

Comment:
Wouldn't the removal of `if lvl <= $crate::STATIC_MAX_LEVEL && lvl <= $crate::max_level()` cause the build time removal of the log entries over a specific level to stop to work?

Reply:
That gets expanded in `log_enabled!` IIRC. So it would be the same.